packages/img/src/getImage.ts

```diff
@@ -1,5 +1,9 @@
 import { Endpoint, EndpointInstance } from '@rest-hooks/endpoint';
 
+function isNode() {
+  return process !== undefined && process.release.name === 'node';
```

Comment:
```suggestion
  return typeof process !== 'undefined' && process.release.name === 'node';
```
Otherwise a broken bundler (like webpack 5) will cause this to throw a reference error.